### ISSUE TITLE
Table with sequences

### ISSUE THREAD

**Issue by sjh37** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#668]
I have a table WITH Sequence SQL Server.
![image](https://user-images.githubusercontent.com/4190514/107971531-7f8cf600-6faa-11eb-8f6b-e20375eacb95.png)

I have the following table and column:
![image](https://user-images.githubusercontent.com/4190514/107971558-89aef480-6faa-11eb-8997-a3f3a1575f0c.png)

After generating the code with the .tt template:

Current result: 
![image](https://user-images.githubusercontent.com/4190514/107971594-96334d00-6faa-11eb-81a0-e48645b21336.png)

Expected result:
![image](https://user-images.githubusercontent.com/4190514/107971620-9d5a5b00-6faa-11eb-96e9-156193b0f9e6.png)

According to the documentation provided by Microsoft,  [it should work like this](https://docs.microsoft.com/en-us/ef/core/modeling/sequences)

**Comment by sjh37** [comment]
```sql
CREATE SEQUENCE dbo.CountBy1 AS INT START WITH 1 INCREMENT BY 1;
GO
SELECT NEXT VALUE FOR dbo.CountBy1;
CREATE TABLE dbo.SequenceTest
(
    Id INT NOT NULL PRIMARY KEY DEFAULT (NEXT VALUE FOR dbo.CountBy1),
    CarMake VARCHAR(20) NOT NULL
);
GO
```

**Comment by estyfen** [comment]
Hi @sjh37 !!

When do you plan to release the version?

Regards

**Comment by sjh37** [comment]
There is one more bit I would like to add and that is adding this line to the `OnModelCreating` function:
```c#
modelBuilder.HasSequence<int>("CountBy1");
```

You can grab the latest code now as you don't need this to get going: Latest file here: [EF.Reverse.POCO.v3.ttinclude](https://raw.githubusercontent.com/sjh37/EntityFramework-Reverse-POCO-Code-First-Generator/master/EntityFramework.Reverse.POCO.Generator/EF.Reverse.POCO.v3.ttinclude)

**Comment by sjh37** [comment]
Closing this case as it will be in the next release.

**Comment by sjh37** [comment]
This has now been released in v3.4.0
